pom.xml

```diff
@@ -297,21 +297,16 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <aspectj.version>1.9.7</aspectj.version>
     <aspectj-plugin.version>1.14.0</aspectj-plugin.version>
     <restrict-imports.enforcer.version>2.4.0</restrict-imports.enforcer.version>
-    <bzip2.version>1.0.8</bzip2.version>
-    <zlib.version>1.2.13</zlib.version>
```

Comment:
nit: we need to remove these checks as well:
https://github.com/apache/ozone/blob/master/hadoop-ozone/dev-support/checks/native.sh#L22-L32